nodes/consensus-node.md

```diff
@@ -199,15 +198,22 @@ pruning = "everything"
 min-retain-blocks = 0 # this is the default setting
 ```
 
-## Syncing
+## Sync types
+
+| Mode              | Block sync  | State sync      | Quick sync |
+|-------------------|-------------|-----------------|------------|
+| Time              | ~3 weeks    | TBD             | TBD        |
+| Trust assumptions | Most secure | Trusts snapshot | TBD        |
```

Comment:
I'm collecting this data now. Should we hold off on merging this table with TBDs? I can add it back after I have the data.

```suggestion
```